xpanse/__init__.py

```diff
@@ -1,6 +1,6 @@
-__version__ = "1.0.12"
+__version__ = "1.1.0"
 __author__ = "Palo Alto Cortex Xpanse"
 __license__ = "ISC"
 __maintainer__ = "Palo Alto Cortex Xpanse"
 __email__ = "help@expanseinc.com"
-__status__ = "Development"
+__status__ = "Production"
```

Comment:
@cachukwu forgot to bump this version, and made the status "Production". I don't think we use this status anywhere in the docs, but updated to keep it consistent with the status of the project.

Reply:
I'm going to merge these changes( only update since your last review)